modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-multiple-bsl-use-case_{context}"]
+= {oadp-short} use case for two BSLs
+
+In this use case, you configure the DPA with two storage locations by using two cloud credentials. You backup an application with a database by using the default BSL. {oadp-short} stores the backup resources in the default BSL. You then backup the application again by using the second BSL.
+
+.Prerequisites
+
+* You must install the {oadp-short} Operator.
+* You must configure two backup storage locations, AWS S3 and Multicloud Object Gateway (MCG).
+* You must have an application with a database deployed on a Red Hat OpenShift Cluster.
+
+.Procedure
+
+. Create the first `Secret` for AWS S3 storage provider with the default name by running the following command.
++
+[source, terminal]
+----
+oc create secret generic cloud-credentials -n openshift-adp --from-file cloud=<aws_credentials_file_name> # <1>
+----
+<1> Specify the name of the cloud credentials file for AWS S3.
+
+. Create the second `Secret` for MCG with a custom name by running the following command.
++
+[source, terminal]
+----
+oc create secret generic mcg-secret -n openshift-adp --from-file cloud=<MCG_credentials_file_name> # <1>
+----
+<1> Specify the name of the cloud credentials file for MCG. Note the name of the custom secret `mcg-secret`.
+
+. Configure the DPA with the two BSLs as shown in the following example.
++
+[source, yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: two-bsl-dpa
+  namespace: openshift-adp  
+spec:
+  backupLocations:
+  - name: aws
+    velero:
+      config:
+        profile: default
+        region: <region_name> # <1>
+      credential:
+        key: cloud
+        name: cloud-credentials
+      default: true
+      objectStorage:
+        bucket: <bucket_name> # <2>
+        prefix: velero
+      provider: aws
+  - name: mcg
+    velero:
+      config:
+        insecureSkipTLSVerify: "true"
+        profile: noobaa
+        region: <region_name> # <3>
+        s3ForcePathStyle: "true"
+        s3Url: <s3_url> # <4>
+      credential:
+        key: cloud
+        name: mcg-secret # <5>
```

Comment:
Should `mcg-secret` be formatted as a user-replaced variable? That is, `<mcg_secret>`.